prow/hook/plugins.go

```diff
@@ -45,6 +45,7 @@ import (
 	_ "k8s.io/test-infra/prow/plugins/size"
 	_ "k8s.io/test-infra/prow/plugins/skip"
 	_ "k8s.io/test-infra/prow/plugins/slackevents"
+	_ "k8s.io/test-infra/prow/plugins/stage"
```

Comment:
This cannot be in the same PR as the plugins.yaml change. It belongs in the PR that creates the plugin.

After we merge the prow/hook/plugins.go change we will need to cut and deploy a new prow release, after which time we can make the plugins.yaml change to enable it.

Reply:
:+1:

Reply:
fixed.